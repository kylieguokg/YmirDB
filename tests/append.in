append a 28
set a 18 29 3
get a
append a 1 2 3
get a
set b 1 20 3
set c 1 b 2
append a c
list entries
forward a
forward b
forward c
backward a
backward b
backward c
bye
