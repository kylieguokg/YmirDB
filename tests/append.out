> no such key

> ok

> [18 29 3]

> ok

> [18 29 3 1 2 3]

> ok

> ok

> ok

> c [1 b 2]
b [1 20 3]
a [18 29 3 1 2 3 c]

> b, c

> nil

> b

> nil

> a, c

> a

> bye
